```
[ant:checkstyle] [ERROR] /home/travis/build/spring-projects/spring-integration/spring-integration-core/src/main/java/org/springframework/integration/dsl/StandardIntegrationFlow.java:128:41: Reference to instance variable 'integrationComponents' needs "this.". [RequireThis]
[ant:checkstyle] [ERROR] /home/travis/build/spring-projects/spring-integration/spring-integration-core/src/main/java/org/springframework/integration/dsl/StandardIntegrationFlow.java:143:41: Reference to instance variable 'integrationComponents' needs "this.". [RequireThis]
```

--- a/spring-integration-core/src/main/java/org/springframework/integration/dsl/StandardIntegrationFlow.java
+++ b/spring-integration-core/src/main/java/org/springframework/integration/dsl/StandardIntegrationFlow.java
@@ -125,7 +125,7 @@ public class StandardIntegrationFlow implements IntegrationFlow, SmartLifecycle 
 	@Override
 	public void stop(Runnable callback) {
 		AggregatingCallback aggregatingCallback = new AggregatingCallback(this.integrationComponents.size(), callback);
-		for (Object component : integrationComponents.keySet()) {
+		for (Object component : this.integrationComponents.keySet()) {
 			if (component instanceof SmartLifecycle) {
 				SmartLifecycle lifecycle = (SmartLifecycle) component;
 				if (lifecycle.isRunning()) {
@@ -140,7 +140,7 @@ public class StandardIntegrationFlow implements IntegrationFlow, SmartLifecycle 
 
 	@Override
 	public void stop() {
-		for (Object component : integrationComponents.keySet()) {
+		for (Object component : this.integrationComponents.keySet()) {
 			if (component instanceof SmartLifecycle) {
 				SmartLifecycle lifecycle = (SmartLifecycle) component;
 				if (lifecycle.isRunning()) {

